master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,152 @@
+---
+title: AWS
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking not only for private data centers, but also within a
+public cloud such as Amazon Web Services (AWS).  The following information
+details necessary modifications to your standard AWS deployment to allow
+Calico to function successfully.
+
+
+## Prerequisite: Host Connectivity
```

Comment:
We should document the exact holes that need to be poked in the AWS security-group in order for Calico to work.

These will vary based on whether you're disabling src/dst check or if you're using `ipip`